e2e_tests/tests/trial/trial.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+import pytest
+
+from determined.common.api import authentication, bindings, certs
+from determined.common.api.bindings import determinedexperimentv1State
+from determined.common.experimental import session
+from tests import config as conf
+from tests import experiment as exp
+
+
+def create_exp_get_trial_id():
```

Comment:
`e2e_tests/tests/experiment/experiment.py` has a lot of similar helpers. maybe put it there for the sake of anyone else being able to find this/use it.